frontend/src/app/components/team/team.component.ts

```diff
@@ -16,16 +19,23 @@ import { DialogService } from '../../services/dialog.service';
 export class TeamComponent implements OnInit {
   @Input({ required: true })
   public overviewEntity!: OverviewEntity;
+  addIconSrc: BehaviorSubject<string> = new BehaviorSubject('assets/icons/new-icon.svg');
 
   constructor(
     private dialogService: DialogService,
     private refreshDataService: RefreshDataService,
-  ) {}
-
-  trackByObjectiveId: TrackByFunction<ObjectiveMin> = (index, objective) => objective.id;
+    private configService: ConfigService,
+  ) {
+    this.configService.config$.subscribe((config: ClientConfig) => {
```

Comment:
this subscribe poses a potential memory leak because it is never properly destroyed